migrations/20221216122421_username_table.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  await knex.schema.createTable('usernames', (table) => {
```

Comment:
@kamicut maybe calling the table `users` as it will hold other metadata about the users?

Reply:
We already have a users table that holds metadata, but it's only for logged in users. This is simply a table for holding the display name. I was thinking of renaming that table to "accounts" and this one to users but it feels it could break a lot of things. What do you think?

Reply:
I renamed to `osm_users` as per our call